.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+package-lock.json
```

Comment:
I think this is potentialy dangerous, package-lock.json must be added to Git.

From [NPM documentation](https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json):
> This file is intended to be committed into source repositories